### PR TITLE
NOISSUE - Configure charts to display other values

### PR DESCRIPTION
**Pull request by darkodraskovic**
Currently, charts only display number values. Enable the display of other types of values, such as booleans, too.

### CODE REVIEW

File: src/app/shared/chart/chart.component.ts [hunk 1]
```diff
@@ -75,4 +74,15 @@ export class ChartComponent implements OnChanges {
       this.chart && this.chart.update();
     });
   }
+
+  findValue(message: MsgResp): any {
```
Comment (manuio) [inline src/app/shared/chart/chart.component.ts:78]: I'm not sure about `find`. Maybe `parseValue` ?

File: src/app/common/interfaces/mainflux.interface.ts [hunk 2]
```diff
@@ -57,3 +57,14 @@ export interface Message {
   v: number;
   vs: string;
 }
+
+export interface MsgResp {
```
Comment (manuio) [inline src/app/common/interfaces/mainflux.interface.ts:61]: Maybe better `MfxMsg` or `MainfluxMsg`

**Review by manuio — CHANGES_REQUESTED** [review]

**Review by manuio — APPROVED** [review]
LGTM

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**